Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs

```diff
@@ -403,7 +403,9 @@ protected bool UpdateHandData(OVRHand ovrHand, OVRSkeleton ovrSkeleton)
             // Disable hand if not tracked
             if (handRenderer != null)
             {
-                handRenderer.enabled = isTracked;
+                MixedRealityHandTrackingProfile handTrackingProfile = CoreServices.InputSystem?.InputSystemProfile.HandTrackingProfile;
```

Comment:
We may want to cache the HandTrackingProfile somewhere, so these scripts aren't reaching into the InputSystem every update.